src/main/java/com/tailf/jnc/NetconfSession.java

```diff
@@ -1756,7 +1755,7 @@ void checkMid(Element t, String mid) throws JNCException {
         RPCRequest()
         {
             msgId = message_id++ ;
-            message = new StringBuilder();
+            message = new StringBuilder(64); // few extra bytes to save a few re-inits
```

Comment:
Can you explain a little more what's going on here? No need for comments in the code, just here in the pull-request.

Reply:
Static analysis reveals that there's some minimum of bytes written to this builder, that is larger than default 16B `StringBuilder`'s buffer, causing unnecessary re-inits/re-allocs...
https://pmd.sourceforge.io/pmd-6.53.0/pmd_rules_java_performance.html#insufficientstringbufferdeclaration